dockerfiles/centos/is-analytics/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+# set to latest Centos
+FROM centos:7
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org”
+
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# set dependant files directory
+ARG FILES=./files
+# set jdk configurations
+ARG JDK=jdk1.8.0*
+ARG JAVA_HOME=${USER_HOME}/java
+# set wso2 product configurations
+ARG WSO2_SERVER=wso2is-analytics
+ARG WSO2_SERVER_VERSION=5.5.0
+ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
+# install required packages
+#RUN yum update  && \
+#  yum -y --downloadonly \
+#  curl \
+#  nc && \
+#  yum clean
+RUN yum check-update && \
+    DEBIAN_FRONTEND=noninteractive yum install --downloadonly \
+    #curl \
+    nc && \
+    rm -rf /var/cache/yum/* && \
+    >> /etc/bash.bashrc \
+    ; echo "\
```

Comment:
EULA should be only included if the Image contains WUM updates right? We might have to figure out a way to solve this since these resources are used for both WUM update and GA images.